main.py

```diff
@@ -1,26 +1,113 @@
+import requests
 import functools
 import os
+import subprocess
 import random
 import sys
 import time
 
+import speech_recognition as sr
 from faker import Faker
 from selenium import webdriver
 from selenium.webdriver.support.ui import Select
+from selenium.webdriver.common.keys import Keys
 
 from constants.common import *
+from constants.fileNames import *
+from constants.classNames import *
 from constants.elementIds import *
 from constants.email import *
 from constants.location import *
 from constants.urls import *
 from constants.xPaths import *
 
+os.environ["PATH"] += ":/usr/local/bin" # Adds /usr/local/bin to my path which is where my ffmpeg is stored
+
 fake = Faker()
 chromedriver_location = CHROMEDRIVER_PATH
 # Change default in module for print to flush
 # https://stackoverflow.com/questions/230751/how-can-i-flush-the-output-of-the-print-function-unbuffer-python-output#:~:text=Changing%20the%20default%20in%20one%20module%20to%20flush%3DTrue
 print = functools.partial(print, flush=True)
 
+r = sr.Recognizer()
+
+def audioToText(mp3Path):
+    # deletes old file
+    try:
+        os.remove(CAPTCHA_WAV_FILENAME)
+    except FileNotFoundError:
+        pass
+    # convert wav to mp3                                                            
+    subprocess.run(f"ffmpeg -i {mp3Path} {CAPTCHA_WAV_FILENAME}", shell=True, timeout=5)
```

Comment:
Are there any benefits to using ffmpeg over pydub's  AudioSegment? Seems like an extra layer of complexity for the end user (installing ffmpeg & setting up the path) vs 3 lines of code in main.py and 1 in requirements.txt à la #26 .
A better way than both methods would be probably be to save the incoming stream straight to WAV, but my brain is too fried to work that out at the moment.
Also, is checking for the file and deleting necessary before running ffmpeg? Using the solution in #26, `sound.export()` overwrites the old WAV file during each new iteration, perhaps ffmpeg can't do that for some reason (permissions?) or needs an additional command flag?

Reply:
I simply couldn't get pydubs working. I think you are right though saving directly to wav would be best

Reply:
Also my understanding is that pydubs relies on ffmpeg for conversion anyway

Reply:
I see. I wonder if anyone else who's tested with the #26 solution has had issues with pydub? I hope not... :facepalm: 
I didn't mean to give you hard time about ffmpeg, but I've done A LOT of scripting with it in the past. And while it's been...a while...for me, ffmpeg used to have a pretty dodgy reputation for breaking things with their updates, in addition to some nasty security loopholes stemming from outdated libraries. Maybe that's not the case anymore, but it was bad enough just a few years ago that I would regularly have to re-write scripts after updates to account for altered internal default values or implementation differences between distros. That's my unsolicited $0.02 on ffmpeg :sweat_smile:

Reply:
> Also my understanding is that pydubs relies on ffmpeg for conversion anyway

Well I learned [something](https://github.com/jiaaro/pydub#dependencies) today. I should have RTFM. Doh!

ffmpeg on its own is still kludgy IMO, maybe that's why I prefer using the pydub wrapper. When I get some time, I'll work out the "clean" solution and hopefully we'll be done with creating multiple audio files.